features/polyExplorer/src/components/buttons/infoButton/infoButton.jsx

```diff
@@ -3,24 +3,22 @@ import LinkButton from "../linkButton/linkButton.jsx";
 import "./infoButton.css";
 
 const InfoButton = ({
+    source,
     light = false,
     route,
     saveActiveIndex,
     stateChange = null,
 }) => (
-    <div onClick={saveActiveIndex}>
+    <div onClick={saveActiveIndex} className="source-info-container">
```

Comment:
I do not think the source should be part of this. The infobutton needs to be alone as a component and then we could make a new component with the infobutton and the source. Shouldn't be as one